pipeline/build/azure-pipelines.yaml

```diff
@@ -55,6 +55,9 @@ stages:
     parameters:
       terraformPath: $(workDir)
       terraformVersion: $(tfVersion)
+      azureDevOpsURL: $(devOpsURL)
+      numberOfAgents: ${{ variables.agentCount }}
+      azureDevOpsPAT: $(patToken)
```

Comment:
I think you are missing defining this variable? patToken

I cant see it defined anywhere, just used

Reply:
this is coming from the var lib, just following how it was done previously.

https://dev.azure.com/hmcts/Video%20Hearings/_library?itemType=VariableGroups&view=VariableGroupView&variableGroupId=371&path=vh-azure-devops-agent

Reply:
ah okay, I missed the group